deploy_dod_dataset.py

```diff
@@ -3,7 +3,7 @@
 import logging
 
 from libs.enums import Intervention
-from libs.pipelines import dod_pipeline
+from libs.pipelines import dod_pipeline, top_counties_pipeline
```

Comment:
best to throw these on separate lines (standard is one import per line), but you probably can actually remove this

Reply:
cool yes done removed